src/components/chat/MessageForm.js

```diff
@@ -90,6 +109,9 @@ class MessageForm extends Component {
                 value={message}
                 ref={(input) => { this.messageInput = input; }}
                 placeholder="Enter message" required></textarea>
+                <Button onClick={this.onSendMessage}>
+                  <FaArrowCircleRight size={30} />
+                </Button>
```

Comment:
@elimisteve now I remember, I just changed the location of the button to be within the `message` class, so we can create a flex container for the chat icons.